mne/viz/_brain/tests/test.ipynb

```diff
@@ -39,16 +42,40 @@
     "initial_time = 0.13\n",
     "mne.viz.set_3d_backend('notebook')\n",
     "brain_class = mne.viz.get_brain_class()\n",
-    "for interactive_state in (False, True):\n",
-    "    plt.interactive(interactive_state)\n",
+    "\n",
+    "\n",
+    "@contextmanager\n",
+    "def interactive(on):\n",
+    "    old = plt.isinteractive()\n",
+    "    plt.interactive(on)\n",
+    "    try:\n",
+    "        yield\n",
+    "    finally:\n",
+    "        plt.interactive(old)\n",
+    "\n",
+    "with interactive(False):\n",
     "    brain = stc.plot(subjects_dir=subjects_dir, initial_time=initial_time,\n",
     "                     clim=dict(kind='value', pos_lims=[3, 6, 9]),\n",
     "                     time_viewer=True,\n",
-    "                     hemi='split')\n",
+    "                     show_traces=True,\n",
+    "                     hemi='split', size=300)\n",
     "    assert isinstance(brain, brain_class)\n",
     "    assert brain.notebook\n",
     "    assert brain._renderer.figure.display is not None\n",
     "    brain._update()\n",
+    "    total_number_of_buttons = len([k for k in brain.actions.keys() if '_field' not in k])\n",
+    "    number_of_buttons = 0\n",
+    "    for action in brain.actions.values():\n",
+    "        if isinstance(action, Button):\n",
+    "            action.click()\n",
+    "            number_of_buttons += 1\n",
+    "    assert number_of_buttons == total_number_of_buttons\n",
+    "    img_nv = brain.screenshot()\n",
+    "    assert img_nv.shape == (300, 300, 3), img_nv.shape\n",
+    "    img_v = brain.screenshot(time_viewer=True)\n",
+    "    assert img_v.shape[1:] == (300, 3), img_v.shape\n",
+    "    # XXX This rtol is not very good, ideally would be zero\n",
+    "    assert_allclose(img_v.shape[0], img_nv.shape[0] * 1.25, err_msg=img_nv.shape, rtol=0.1)\n",
```

Comment:
HiDPI aside, on Linux I was getting values that were off by ~7%. It would be good if this were as good as the non-notebook backend (which uses `atol=3` instead of the `rtol=0.1` here).